setup.sh

```diff
@@ -100,6 +100,7 @@ main() {
 
   if [ -n "${SECRET_NAME}" ]; then
     add_secret_name_accessor_permission
+    REQUIRED_SERVICES += ('Secret Manager API')
```

Comment:
This is not correct: this code would set the SM permission and then check that the SM API is enabled. You need to check that the API is enabled first and then set the SM permission.